apps/passport-client/components/screens/HomeScreen.tsx

```diff
@@ -29,11 +26,13 @@ export function HomeScreen() {
     }
   });
 
-  const cards = useMemo(
-    () => getTestCards(state.identity, state.self),
-    [state]
-  );
-  const [sel, _setSel] = useState(0);
+  const zuzaluPCDId = useMemo(() => {
+    return state.pcds.getAll()[0].id;
+  }, [state.pcds]);
+  const [selectedPCDID, setSelectedPCDID] = useState(() => zuzaluPCDId);
```

Comment:
nit but this can be just `= useState(zuzaluID)`

& i'd be consistent (`PCDID` vs `PCDId` => maybe just `ID`?)